admin/partials/page-theme-edit.php

```diff
@@ -21,6 +21,7 @@
 			<div class="apple-news-settings-left">
 				<h3><?php esc_html_e( 'Theme Settings', 'apple-news' ); ?></h3>
 				<p><?php esc_html_e( 'Configuration for the visual appearance of the theme. Updates to these settings will not change the appearance of any articles previously published to your channel in Apple News using this theme unless you republish them.', 'apple-news' ); ?></p>
+				<p><?php esc_html_e( 'Apple News supports Dark Mode.  Each applicable component is now given the ability to set its own dark mode colors.  This will override the Smart Defaults that Apple News will attempt to use.', 'apple-news' ); ?></p>
```

Comment:
```suggestion
				<p><?php esc_html_e( 'Apple News supports Dark Mode. Each applicable component is now given the ability to set its own dark mode colors. This will override the intelligent defaults that Apple News will attempt to use.', 'apple-news' ); ?></p>
```